src/components/ReportActionItem/IOUAction.js

```diff
@@ -47,7 +47,7 @@ const IOUAction = (props) => {
             {((props.isMostRecentIOUReportAction && Boolean(props.action.originalMessage.IOUReportID))
                 || (props.action.originalMessage.type === 'pay')) && (
                     <IOUPreview
-                        iouReportID={props.action.originalMessage.IOUReportID}
+                        iouReportID={props.action.originalMessage.IOUReportID.toString()}
```

Comment:
Q: Shouldn't we change the IOUReportID property to string instead?

Reply:
Where are you thinking we need to make this change? I think there was some kind of migration and noticed that all reportID are assumed to be strings and that errors in the JS console showed propType warnings so I casted this to a string. Maybe @tgolen has thoughts on how to respond.

Reply:
When I looked at this, I found that the PHP APIs that returned this data were still using the OldDot API. I made the assumption that once those APIs got migrated over to be NewDot APIs, then we could make the change on the PHP side. If those haven't been migrated yet, I think it's fine to cast these to strings for now, but it should be cleaned up once PHP is able to send them as strings.

Reply:
Cool, that makes sense to me then! I guess this is not really hurting anything for now. 

Is there a tracking issue for that migration or anything I can link a clean up issue to floating around?

Reply:
Nothing for the IOU reportIDs yet, no.